PS4k/src/com/vitco/core/EngineViewPrototype.java

```diff
@@ -86,9 +86,28 @@ public final int getWidth() {
         return container.getWidth();
     }
     // get the image rendered in container in high quality
-    public BufferedImage getImage() {
+    public BufferedImage getImage() throws Exception {
```

Comment:
Can you make the Exception that is thrown explicit then? I.e. IOException instead of Exception?